resources/test/bedcounts.csv

```diff
@@ -1,2 +1,2 @@
 icu_name,n_covid_occ,n_covid_free,n_ncovid_occ,n_ncovid_free,n_covid_deaths,n_covid_healed,n_covid_refused,n_covid_transfered,create_date
-hopital3,11,12,13,14,15,16,17,18,2020-04-16 23:00
\ No newline at end of file
+hopital3,11,12,13,14,15,16,17,18,2020-04-16 23:00+02
```

Comment:
We control the format of CSV right? So we add the timezone in the csv as needed?

Reply:
Yes, AFAIK nobody relies on the format yet.